src/shared/components/plots/StackedBarPlot.tsx

```diff
@@ -358,11 +312,12 @@ export default class StackedBarPlot extends React.Component<IStackedBarPlotProps
 
     @computed get chartExtent() {
         const miscPadding = 100; // specifying chart width in victory doesnt translate directly to the actual graph size
-        const numBars = this.data.length;
-        return this.categoryCoord(numBars - 1) + 2*this.categoryAxisDomainPadding + miscPadding;
-        //return 2*this.domainPadding + numBoxes*this.barWidth + (numBoxes-1)*this.barSeparation;
-        //const ret = Math.max(computedExtent, this.props.chartBase);
-        //return ret;
+        if (this.data.length > 0) {
+            const numBars = this.data[0].counts.length;
```

Comment:
I don't think testing this makes sense - it is subject to the specific hardcoded aesthetic values for padding. I think this is the kind of thing that validation testing should catch - whether the chart looks right. Our existing screenshot tests catch this wonky behavior but I/we didn't pay close enough attention. Our updated screenshots should catch it.

Reply:
sounds good